tests/unit/utils/SGObjectIterator.h

```diff
@@ -0,0 +1,164 @@
+#ifndef __SG_OBJECT_ITERATOR_H__
+#define __SG_OBJECT_ITERATOR_H__
+
+#include <iterator>
+#include <set>
+#include <string>
+#include <shogun/base/class_list.h>
+
+using namespace shogun;
```

Comment:
@theartful plz never do this in a header file...

Reply:
my bad didn't see it

Reply:
ouch! sorry for that

Reply:
@vigsterkr https://github.com/shogun-toolbox/shogun/blob/79fa98a85aed7eb88be30884349dd0666567ac5a/tests/unit/utils/SGObjectIterator.h#L9
shouldn't it be namespace shogun?